src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types'; /* eslint-disable react/forbid-prop-types */
+import Display from './Display';
+import Keypad from './Keypad';
+import Operations from './Operations';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <Display />
+        <Keypad digits={this.props.nums} />
+        <Operations ops={this.props.ops} />
+      </div>
+    );
+  }
+}
+
+Calculator.propTypes = {
+  nums: PropTypes.array.isRequired,
+  ops: PropTypes.array.isRequired,
+};
+
+export default Calculator;
```

Comment:
- [ ] Kindly stick to the design guideline as your current design does not match
  Your design :point_down: 
  
![Screenshot from 2022-05-25 13-26-06](https://user-images.githubusercontent.com/66097371/170262203-0958a848-d093-4c0e-b2b1-36a3aeeab596.png)
   Design Guidline :point_down: 
    
![Screenshot from 2022-05-25 13-26-54](https://user-images.githubusercontent.com/66097371/170262345-4d06c98e-54e9-4f25-ba38-60a988655c38.png)